liferay-echo/liferay-portal
# LPS-182317 Adds ootb utility page 500 to welcome site initializer

## Motivation

[Link to ticket](https://issues.liferay.com/browse/LPS-184381)

This PR adds the files needed to have an error 500 utility page in the welcome site initializer

## Change summary:

* Adds utility page and page definitions
* Adds new images


## Test Scenario

* Deploy the code in the PR
* Create a new site based on welcome site initializer
* Go to Pages -> Utility Pages tab
* Utility Page for error 500 should be present with error 404
